include/picongpu/CMakeLists.txt

```diff
@@ -130,15 +130,15 @@ set(CMAKE_CXX_STANDARD 17)
 get_filename_component(SOURCE_DIR_ROOT ${PIConGPUapp_SOURCE_DIR}/../.. ABSOLUTE)
 string(FIND "${PIConGPUapp_BINARY_DIR}"
             "${SOURCE_DIR_ROOT}/" IN_SRC_POS)
-if(IN_SRC_POS GREATER -1)
-  message(FATAL_ERROR
-    "PIConGPU requires an out of source build. "
-    "Please remove \n"
-    "  - CMakeCache.txt\n"
-    "  - CMakeFiles/\n"
-    "and create a separate build directory. "
-    "See: INSTALL.rst")
-endif()
+#if(IN_SRC_POS GREATER -1)
+#  message(FATAL_ERROR
+#    "PIConGPU requires an out of source build. "
+#    "Please remove \n"
+#    "  - CMakeCache.txt\n"
+#    "  - CMakeFiles/\n"
+#    "and create a separate build directory. "
+#    "See: INSTALL.rst")
+#endif()
```

Comment:
you should not change this, it is fine for your pull request, but not for the PIConGPU in general

Reply:
Yeah, this should not get merged here. However, some IDEs like Visual Studio and CLion, when they splot a CMakeLists.txt, start to configure the project in a subfolder of the working copy, so they can offer better help to the developer. This is still an out-of-source build btw. and you can build picongpu fine this way as well. It is just not the intended workflow when working on a cluster.